cli/internal/cache/cache.go

```diff
@@ -122,16 +123,19 @@ func newSyncCache(opts Opts, repoRoot turbopath.AbsoluteSystemPath, client clien
 	cacheImplementations := make([]Cache, 0, 2)
 
 	if useFsCache {
-		implementation, err := newFsCache(opts, recorder, repoRoot)
+		implementation, err := newFsCache(opts, recorder, base.RepoRoot)
 		if err != nil {
 			return nil, err
 		}
 		cacheImplementations = append(cacheImplementations, implementation)
 	}
 
 	if useHTTPCache {
+		base.LogInfo("• Remote computation caching enabled")
```

Comment:
I feel like the "computation" here is confusing and could be read as "remote computation". I think dropping "Remote caching enabled" is cleaner.

Reply:
I agree! updated